docs/install-pro.md

```diff
@@ -0,0 +1,101 @@
+# Install Percona XtraBackup Pro
+
+--8<--- "pro-build-announcement.md"
+
+This document provides guidelines how to install Pro packages of Percona XtraBackup from Percona repositories. [Check files in packages built for Percona XtraBackup Pro :material-arrow-right:](pro-files.md){.md-button}
+
+## Procedure
+
+1. Request the access to the pro repository from Percona Support. You will receive the client ID and the access token which you use when downloading the packages.
+
+3. Configure the repository and install Percona XtraBackup packages
+
+    === "On Debian and Ubuntu"
```

Comment:
Should it be
On Debian or Ubuntu
? 
Same for RHEL or derivatives

Reply:
fixed